.github/workflows/rnmobile-ios-runner.yml

```diff
@@ -2,16 +2,18 @@ name: React Native E2E Tests (iOS)
 
 on:
     pull_request:
+        type: [labeled, opened, synchronize, reopened]
```

Comment:
`labeled` ensures the workflow is only executed on labelled PRs, the other three types of PRs I've added appear to be defaults which I'm not sure are needed explicitly now that I'm customizing this: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#issues